packages/libsql/test/libsql.test.ts

```diff
@@ -529,13 +546,23 @@ describe("AccessStrategy.Snapshot", () => {
     await service2.read(cartId)
     assertSpans(
       { name: "Transact", [Tags.loaded_count]: 0, [Tags.append_count]: 10 },
-      { name: "Query", [Tags.loaded_from_version]: "10", [Tags.loaded_count]: 0, [Tags.cache_hit]: true },
+      {
+        name: "Query",
+        [Tags.loaded_from_version]: "11",
+        [Tags.loaded_count]: 0,
+        [Tags.cache_hit]: true,
+      },
     )
     memoryExporter.reset()
 
     // Add two more - the roundtrip should only incur a single read
     await CartHelpers.addAndThenRemoveItemsManyTimes(cartContext, cartId, skuId, service1, 1)
-    assertSpans({ name: "Transact", [Tags.loaded_count]: 0, [Tags.append_count]: 2, [Tags.cache_hit]: true })
+    assertSpans({
```

Comment:
maybe test code should have longer line len
maybe all code should have longer line len
maybe I should go write golang and learn tolerance

Reply:
uuurgh, yeah, fine... Unprettified the file and as expected it reads much better

Reply:
was only lamenting - it's really not the worst